public/humans.txt

```diff
@@ -5,75 +5,87 @@ Oak National Academy’s tech platform has been created by a team of human devel
 /* TEAM */
 
 
+  John Roberts (Director of Product and Engineering)
+  Site: https://johnroberts.me
+  LinkedIn: https://linkedin.com/in/johnrobertsuk
+  Twitter: @johnrobeds
+  From: Manchester, UK
+
   Ian Kynnersley (Head of Engineering)
   Site: https://www.iankynnersley.co.uk/
   LinkedIn: https://www.linkedin.com/in/iankynnersley/
   Twitter: https://twitter.com/kpopper
   From: London, UK
   
+  Andrew Woods (Software Engineering Manager)
+  Site: https://andrewwoods.super.site/
+  LinkedIn: https://www.linkedin.com/in/andykwoods/
+  From: Dublin, Ireland
 
-  Jack Skerrett (QA)
+  Jack Skerrett (Quality Assurance Engineer)
   LinkedIn: https://www.linkedin.com/in/jack-skerrett-73769417a/
   From: Cornwall, UK
 
-
-  Jim Cresswell (Developer)
+  Jim Cresswell (Principal Software Engineer)
   Site: https://jimcresswell.net/
   LinkedIn: https://uk.linkedin.com/in/jimcresswell/
   From: UK
 
-
-  Joe Baker (Developer)
+  Joe Baker (Software Engineer)
   LinkedIn: https://www.linkedin.com/in/joe-baker-96a109219/
   From: London, UK
 
+  Ben Protheroe (Software Engineer)
+  From London, lives in Cornwall
```

Comment:
Needs a :

Reply:
that is fixed in this commit
https://github.com/oaknational/Oak-Web-Application/pull/1659/commits/12786b229b677336cf6e632e8308d27e01b06d07